packages/aws-rfdk/lib/deadline/scripts/bash/installDeadlineRepository.sh

```diff
@@ -132,7 +132,7 @@ if [ ! -z "${SECRET_MANAGEMENT_ARN+x}" ]; then
     exit 1
   fi
   echo "Secret management is enabled. Credentials are stored in secret: $SECRET_MANAGEMENT_ARN"
-  SECRET_MANAGEMENT_ARG="--installSecretsManagement true --secretsAdminName \"$SECRET_MANAGEMENT_USER\" --secretsAdminPassword \"$SECRET_MANAGEMENT_PASSWORD\""
+  SECRET_MANAGEMENT_ARG="--installSecretsManagement true --secretsAdminName $SECRET_MANAGEMENT_USER --secretsAdminPassword $SECRET_MANAGEMENT_PASSWORD"
```

Comment:
Were the double quotes causing problems here? I'm worried that the password requires special characters in it and without surrounding it in quotes bash could end up interpreting them in unintended ways

Reply:
Yeah, turns out the escaped double quotes within a double-quoted bash string get treated literally. When we use this variable in the call to the repository installer. I've updated this to use bash arrays which will give us the result we want (bash won't treat the quotes literally like before and any whitespace inside a parameter value will be respected)